scripts/webify.java

```diff
@@ -0,0 +1,224 @@
+        System.exit(exitCode);
+    }
+
+
+    @Override
+    public Integer call() throws Exception {
+        System.out.println(this);
+        this.outdir = this.destdir.resolve("documentation").resolve(this.projectVersion).resolve("html");
+        this.dataDestPath = this.destdir.resolve("_data/documentation").resolve(this.projectVersion.replace(".", "_") + ".yaml");
+
+        FileSystem fs = FileSystems.getDefault();
+        var tocifyGlob = fs.getPathMatcher("glob:" + this.tocifyGlob);
+        var omitGlobs = this.omitGlobs.stream().map(glob -> fs.getPathMatcher("glob:" + glob)).toList();
+        var datafyGlob = fs.getPathMatcher("glob:" + this.datafyGlob);
+
+        walk(omitGlobs, tocifyGlob, datafyGlob);
+
+        Files.writeString(outdir.getParent().resolve("index.md"), """
+---
+layout: released-documentation
+title: Documentation
+---
+        """, StandardOpenOption.CREATE, StandardOpenOption.TRUNCATE_EXISTING);
+
+        return 0;
+    }
+
+    private void walk(List<PathMatcher> omitGlobs,
+                        PathMatcher tocifyGlob,
+                        PathMatcher datafyGlob) throws IOException {
+        var resultRootObject = this.mapper.createObjectNode();
+        var resultDocsArray = resultRootObject.putArray("docs");
+        try (var stream = Files.walk(this.srcDir)) {
+            stream.forEach((Path filePath) -> {
+                try {
+                    if (!Files.isRegularFile(filePath)) {
+                        return;
+                    }
+                    var relFilePath = this.srcDir.relativize(filePath);
+                    var outFilePath = this.outdir.resolve(relFilePath);
+                    var omitable = omitGlobs.stream().anyMatch(glob -> glob.matches(relFilePath));
+                    var tocifiable = tocifyGlob.matches(relFilePath);
+                    var datafiable = datafyGlob.matches(relFilePath);
+                    if (omitable && !tocifiable && !datafiable) {
+                        return;
+                    }
+                    else if (!omitable && tocifiable && !datafiable) {
+                        tocify(filePath, outFilePath);
+                    }
+                    else if (!omitable && !tocifiable && datafiable) {
+                        System.out.println("datafiable " + filePath);
+                        var dataDocObject = addToMetadata(filePath, relFilePath, resultDocsArray);
+                        System.out.println("datafiable " + dataDocObject);
+                        if (!dataDocObject.has("path")) {
+                            Files.createDirectories(outFilePath.getParent());
+                            Files.writeString(outFilePath.getParent().resolve("index.html"),
+                                    """
+---
+layout: guide
+title: ${doc.name}
+description: ${doc.description}
+release: ${project.version}
+---
+                                            """.replace("${doc.name}", dataDocObject.get("name").textValue())
```

Comment:
Given we have about a million arguments now what's the harm in adding a couple more to specify liquid template/frontmatter files ?